src/Microsoft.TestPlatform.Extensions.TrxLogger/Utility/Converter.cs

```diff
@@ -509,15 +520,25 @@ private static IList<string> ToResultFiles(ObjectModel.AttachmentSet attachmentS
 
                 string sourceFile = uriDataAttachment.Uri.LocalPath;
                 Debug.Assert(Path.IsPathRooted(sourceFile), "Source file is not rooted");
-
                 // copy the source file to the target location
                 string targetFileName = FileHelper.GetNextIterationFileName(testResultDirectory, Path.GetFileName(sourceFile), false);
-                CopyFile(sourceFile, targetFileName);
 
-                // Add the target file name to the result files list.
-                // (Trx viewer automatically adds In\<Guid> to the result file.
-                string fileName = Path.Combine(Environment.MachineName, Path.GetFileName(targetFileName));
-                resultFiles.Add(fileName);
+                try
```

Comment:
Did we hit any while validating?

Reply:
Related to issue #1705 TRX Logger shouldn't fail if test result attachments are missing/Invalid.